### PR TITLE
Add historical data visualization

### PR DESCRIPTION
**Pull request by pratid**
#### Reviewers

@flopezag @jesuspg
#### Description

Show the evolution in time of the sanity status of the different nodes. For that purpose, a new page "Historical" has been added to Infographics.

As current backend doesn't fully support the query for historical data, a sample test JSON has been included and pseudorandom status values are simulated for the nodes. To test this new feature, Rails server must be run for test environment:

```
$ bin/rails server -e test
```

For clarity, only one region in shown, when clicked, at the same time. Anyway, it's quite easy to compare different regions just by considering the "amount of green" in their respective historical charts. In a future, an average value of node "greenness" could be calculated from a sample of the latest n measures, and shown in the map above. This would give the desired overall, cross-node view.
#### Testing

Verified locally.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,6 @@
 
 # Ignore cache
 /Gemfile.lock
+
+# Ignore IDE files
+/.idea/
```
Comment (flopezag) [inline .gitignore]: add new line

File: test/assets/historical.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      "novaServiceStatus": {"value":"red","description":"desc"},
+      "neutronServiceStatus": {"value":"red","description":"desc"},
+      "cinderServiceStatus": {"value":"red","description":"desc"},
+      "glanceServiceStatus": {"value":"red","description":"desc"},
+      "KPServiceStatus": {"value":"red","description":"desc"},
+      "OverallStatus": {"value":"red","description":"desc"},
+      "FiHealthStatus": {"value":"red","description":"desc"}
+    }
+  ]
+}
```
Comment (flopezag) [inline test/assets/historical.json:44]: You should not add the data for the yellow and blue status of the fi-health?

File: .gitignore (hunk 1 above)
Comment (pratid) [inline .gitignore]: :+1:

File: test/assets/historical.json (hunk 2 above)
Comment (pratid) [inline test/assets/historical.json:44]: Just to clarify:
- FiHealth sanity checks results are `OK`, `NOK` and `POK` (see [sanity_tests.js](https://github.com/telefonicaid/fiware-health/blob/master/fiware-region-sanity-tests/resources/parsers/sanity_tests.js)).
- Infographics backend transforms such values, as well as those from the rest of monitoring checks, into four "color" values: `green`, `red`, `yellow` and `gray`. Therefore, sample historical data simulates those color values.

Given that the mapping between sanity status <-> colors (or between ODC results <-> colors) was not clear enough, some constants have been added to explicit such mappings.

**Comment by jesuspg** [comment]
LGTM

File: test/assets/historical.json (hunk 2 above)
Comment (flopezag) [inline test/assets/historical.json:44]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**